### ISSUE TITLE
[Gameplay/New Feature] Add moving platforms

### ISSUE THREAD

**Issue by ribbybibby** [ribbybibby/Grimm-Journey#13]
A number of Laura's level designs use moving platforms. We didn't have time to implement these for LD31, but we should try and get them in for Leftfield. 

I had a go at doing this before Xmas but found that the level load-in screwed with the way I was translating the objects. Will have to come back to it once the higher priority stuff is knocked on the head.

**Comment by ribbybibby** [comment]
Before I start putting levels together, I'm going to give this a go because I think it would add an extra dimension to the levels (if used sparingly). I'll try to work out a system that allows us to have side to side and up and down platforms.

**Comment by ribbybibby** [comment]
Implemented in 782c37086f0b31ac3cdf276ee5051d36586956f7

I've created a moving platform prefab and a PlatformMovement script to control it. I also updated the BBWController and LRRHController scripts so the two characters play nice with the unusual platforms. 

To use the moving platform prefab, just set the bounds of its movement on the X or Y axis and set the upAndDown bool accordingly (true if up/down; false if left/right). 

If we want to allow even crazier movements, I could probably add some more complexity but I think what we have does for now.

Closed.
